Essentials/src/com/earth2me/essentials/EssentialsConf.java

```diff
@@ -392,11 +392,11 @@ public void setProperty(final String path, final ItemStack stack) {
         set(path, map);
     }
 
-    public void setProperty(String path, List object) {
+    public void setProperty(String path, List<?> object) {
```

Comment:
Never seen this used before. Is it something in a newer version of Java that the IDE complains about now?